### ISSUE TITLE
Adding new tunables to kstat

### ISSUE THREAD

**Issue by imtiazdc** [openzfsonwindows/ZFSin#317]
@lundman Apparently, the metaslabs are unloaded from RAM if they are not used in last 8 transaction groups. If we increase the limit from 8 to 256 it seems to give better performance. We are planning to make it tunable and conduct more experiments. Any recommendations or guidelines we need to follow to make it tunable?

**Comment by lundman** [comment]
Good idea - edit kstat_windows.h, and zfs_kstat_windows.c. Make sure the placement in the struct is the same between the files. Stick with the `KSTAT_DATA_UINT64` as that definitely works (I just rely on casting to handle any conversion). The other types should work, just not as heavily tested.
That's all the changes needed. Basically 2 files, 3 places.

**Comment by datacore-yash** [comment]
@lundman Do you know a way to find the memory usage used exclusively by metaslabs ?
